src/Controls/src/Core/DragAndDrop/DragEventArgs.cs

```diff
@@ -1,21 +1,51 @@
 #nullable disable
 using System;
+using Microsoft.Maui.Graphics;
 
 namespace Microsoft.Maui.Controls
 {
-	/// <include file="../../../docs/Microsoft.Maui.Controls/DragEventArgs.xml" path="Type[@FullName='Microsoft.Maui.Controls.DragEventArgs']/Docs/*" />
+	/// <summary>
+	/// Provides data for the <see cref="DropGestureRecognizer.DragOver"/> and <see cref="DropGestureRecognizer.DragLeave"/> events.
+	/// </summary>
 	public class DragEventArgs : EventArgs
 	{
-		/// <include file="../../../docs/Microsoft.Maui.Controls/DragEventArgs.xml" path="//Member[@MemberName='.ctor']/Docs/*" />
+
+		/// <summary>
+		/// Initializes a new instance of the <see cref="DragEventArgs"/> class.
+		/// </summary>
+		/// <param name="dataPackage">The data package associated with the drag source.</param>
 		public DragEventArgs(DataPackage dataPackage)
 		{
 			Data = dataPackage;
 			AcceptedOperation = DataPackageOperation.Copy;
 		}
 
-		/// <include file="../../../docs/Microsoft.Maui.Controls/DragEventArgs.xml" path="//Member[@MemberName='Data']/Docs/*" />
+		/// <summary>
+		/// Initializes a new instance of the <see cref="DragEventArgs"/> class.
+		/// </summary>
+		/// <param name="dataPackage">The data package associated with the drag source.</param>
+		/// <param name="position">The current location in the coordinate system of the drag.</param>
+		public DragEventArgs(DataPackage dataPackage, Point position)
+		{
+			Data = dataPackage;
+			Position = position;
+			AcceptedOperation = DataPackageOperation.Copy;
```

Comment:
It seems that you are setting the default value of the property to copy in several of the constructors, why not set the default in the property declaration (see comment bellow)? that way we ensure we do not make mistakes if other constructors are added.